### ISSUE TITLE
[bug ?] broken types for gocardless-nodejs/webhooks

### ISSUE THREAD

**Issue by dpr-dev** [gocardless/gocardless-nodejs#110]
Hi there. I'm trying to build integration with gocardless and validate a webhook signature with `parse` function form the `import { parse } from 'gocardless-nodejs/webhooks'`. But my linter doesn't see **webhook.d.ts** and fails with error `File '/...path/node_modules/gocardless-nodejs/webhooks.d.ts' is not a module`. 

How can I include types from the **webhook.d.ts** ?  

Stack: 
- serverless
- typescript

Tsconfig 
```
{
  "compilerOptions": {
    "sourceMap": true,
    "target": "es2020",
    "lib": ["es2020"],
    "strict": true,
    "noUnusedLocals": true,
    "moduleResolution": "node",
    "esModuleInterop": true,
    "typeRoots": ["./node_modules/@types", "./src/types"],
    "outDir": "dist"
  }
}

```

**Comment by sebastien-lb** [comment]
I think this is because `module.exports = {}` is used instead of `export { ... }` in the webhook.ts file. 
A quick fix is to declare the module yourself to be able to import it. 
`declare module "gocardless-nodejs/webhooks"` more on that here:  https://stackoverflow.com/questions/44058101/typescript-declare-third-party-modules

**Comment by szastupov** [comment]
Sorry for the lack of activity, everyone. This should be fixed in #122
